includes/wc-payment-api/models/class-wc-payments-api-charge.php

```diff
@@ -476,6 +476,7 @@ public function jsonSerialize(): array {
 			'refunded'               => $this->get_refunded(),
 			'refunds'                => $this->get_refunds(),
 			'status'                 => $this->get_status(),
+			'captured'               => $this->is_captured(),
```

Comment:
Can we move it before 'refunded' please?

Reply:
Done here [c5b2e48](https://github.com/Automattic/woocommerce-payments/pull/5232/commits/c5b2e483f969fcfbce322c23edea33a2ebdd86d0)